modInterface.py

```diff
@@ -2,6 +2,8 @@
 from modConnection import *
 import Queries
 from modAddProduct import *
+from modAddCred import *
+from modUpdateProd import *
```

Comment:
I didn't see the addition of this file anywhere. Did it just not get included in the commit?